ui/src/components/ReportErrorAlert.tsx

```diff
@@ -1,35 +1,34 @@
 import React from 'react';
 import {Alert, AlertVariant} from '@patternfly/react-core';
 import {hasSignUpTab, uppercaseFirstLetter} from '../utils/utils';
-import {Report} from '../api/report';
+import {ProviderStatus, Report} from '../api/report';
 
 export const ReportErrorAlert = ({report}: { report: Report }) => {
 
   const errorReports = Object.keys(report.providers)
     .map(name => {
       return report.providers[name].status;
     })
-    .filter(e => !e.ok && !hasSignUpTab(e));
+    .filter(e => (!e.ok || Object.keys(e.warnings).length > 0) && !hasSignUpTab(e));
+  
+  const getMessage = (e: ProviderStatus) => {
+    let message = e.message;
+    if(e.ok && Object.keys(e.warnings).length > 0) {
+      return `${uppercaseFirstLetter(e.name)}: ${Object.keys(e.warnings).length} package(s) could not be analyzed`;
+    }
+
+    return `${uppercaseFirstLetter(e.name)}: ${message}`;
+  }
 
   return (
     <>
       {errorReports.map((e, index) => {
-        if(e.name === 'trusted-content') {
-          return <Alert
-          key={index}
-          variant={
-            AlertVariant.info
-          }
-          title={`${uppercaseFirstLetter(e.name)}: Recommendations and remediations are not currently available`}
-        />
-
-        }
         return <Alert
           key={index}
           variant={
             e.code >= 500 ? AlertVariant.danger : e.code >= 400 ? AlertVariant.warning : undefined
           }
```

Comment:
I check the `reportWithWarning.mock.ts`. The warning color style is wrong:


 The logic should be updated to something like:
```suggestion
               variant={
    e.code >= 500 ? AlertVariant.danger :
    e.code >= 400 ? AlertVariant.warning :
    (e.ok && Object.keys(e.warnings).length > 0) ? AlertVariant.warning :
    undefined
  }
```

Reply:
It was left like this intentionally but maybe it's better to use warning in this case as well. Thank you.